components/suggestion/search_date_suggestion/search_date_suggestion.tsx

```diff
@@ -3,67 +3,89 @@
 
 import React from 'react';
 
-import DayPicker from 'react-day-picker';
+import {DayPicker} from 'react-day-picker';
 
-import MomentLocaleUtils from 'react-day-picker/moment';
+import type {Locale} from 'date-fns';
```

Comment:
I swear `react-day-picker` is part of why `moment` crept its way into our code base to begin with, so that change is kind of annoying. I agree that it might be nice to look for a different library for this at some point, but at the very least, because we're dynamically loading these locales, I don't think this should impact bundle size until these components are actually used.

I'd keep an eye on this, but I think it's probably okay.